content/vscode/remote.md

```diff
@@ -0,0 +1,134 @@
+---
+title: มา Dev กันแบบไม่ต้องลง SDK กันเถอะ (Developing inside a Container) 
+showMetadata: true
+editable: true
+showToc: true
+---
+
+
+
+# หากคุณมีปัญหาหรือรู้สึกเบื่อเหนื่อยกับเรื่องเหล่านี้
+
+- เบื่อกับการเตรียม environment ในการ dev หรือต้องมานั่งทำเอกสารเพื่อบอก team ว่าต้องใช้ Nodejs v. ไหน หรือ dotnet sdk version ไหน
+- หรือเปลี่ยนเครื่องคอมใหม่แต่เหนื่อยกับการไล่ลง SDK ที่ต้องใช้ทำงาน
+- อยากลอง SDK ใหม่ๆ แต่ก็กลัวว่าจะมีปัญหากับ project ที่ทำงานอยู่
+- เพื่อนในทีมไม่ได้แม่นเรื่อง Docker แต่อยากให้เขาได้เริ่มใช้งาน
+- อยากให้คนในทีมใช้ extensions ใน vscode ใน project เหมือนๆกัน
+
+# dotnetthailand ขอเสนอ [Remote Development extension](https://marketplace.visualstudio.com/items?itemName=ms-vscode-remote.vscode-remote-extensionpack)
+
+สิ่งที่ต้องเตรียมก็มี (เครื่องเปล่าเปิดซิงก็ทำตามได้ หรือมีอะไรลงไปแล้วก็ข้ามๆไป)
+
+- ลง [Docker](https://docs.docker.com/get-docker/) ก่อนเลย
+- แน่นอนลง [VS Code](https://code.visualstudio.com/download)
+- เสร็จแล้วลง [Remote Development extension](https://marketplace.visualstudio.com/items?itemName=ms-vscode-remote.vscode-remote-extensionpack)
+- แค่นั้นแหละจบ
+
+# ก่อนไปลุยทำความเข้าใจมันนิดว่ามันทำงานยังไง
+
+![alt text][architecture]
+
+ref: https://code.visualstudio.com/docs/remote/containers
```

Comment:
Very minor, I think if we threat ref as a short form or abbreviation, we should use it as `REF`.